### DISCUSSION TITLE
Docs V2

### DISCUSSION THREAD

**Discussion by dickermoshe** [rrousselGit/riverpod#2882] [General]
The old version 2 of the docs was great.
Why was that removed and replaced with a half empty site?

**Comment by rrousselGit** [comment]
It's a dev-release site. It's supposed to change, and is what people wanted when I asked.

What is missing in your opinion from the old site in the new one?

**Reply by dickermoshe** [reply]
There were a bunch of guides, I can't give any specific examples cuz the sites not up.
What commit was the latest one that has the old docs?
I would rebuild and run locally.
Thanks for taking the time! Thanks so much!

**Comment by bowenjin** [comment]
Hey Remi, even though the new V2 docs are incomplete, I'm really liking the new content. I think it introduces concepts in a more understandable order. Also I love the additional detail on how things such as caching work under the hood. Just wondering if you have a timeline as to when the new V2 of the docs will be completed?
